Lab 2.ipynb

```diff
@@ -0,0 +1,1490 @@
+      "text/plain": [
+       "<Figure size 576x576 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "X = 3.7\n",
+    "Y = 7.2\n",
+    "N = 50\n",
+    "\n",
+    "k = np.arange(0,20,1)\n",
+    "original = stats.poisson.pmf(k, X, loc = 0)\n",
+    "final = original\n",
+    "\n",
+    "for i in range(N):\n",
+    "    final = np.convolve(final, original)\n",
+    "\n",
+    "k = np.arange(0, len(final), 1)\n",
+    "\n",
+    "fig, ax = plt.subplots(1, 1, figsize=(8,8))\n",
+    "\n",
+    "plt.tick_params(labelsize = 12)\n",
+    "ax.set_xlim([0,400])\n",
+    "ax.set_ylabel('Probability', fontsize = 12)\n",
+    "ax.set_xlabel('Number of Gamma-rays')\n",
+    "ax.set_title(\"Gamma-ray Occurrences after \" + str(N) + \" Days\")\n",
+    "\n",
+    "ax.step(k, final, 'o-',where = 'mid')\n",
+    "\n",
+    "prob = 0\n",
+    "for i in range(len(k) - int(Y*N)):\n",
+    "    prob = prob + final[int(Y*N) + i]\n",
+    "#np.sum()\n",
+    "    \n",
+    "print('The sigma for a Y of ' + str(Y) + ' is ' + str(stats.norm.ppf(prob, loc = 0, scale = 1)) + '.')"
```

Comment:
Could do abs(sigma), it's symmetric so it doesn't matter but a negative measurement significance doesn't make much sense